### DISCUSSION TITLE
Nominatim OSM latitude and longitude pin

### DISCUSSION THREAD

**Discussion by knee11** [osm-search/Nominatim#2781] [Geocoding around the world]
Hi Everyone,

So we have created a project for our system to run a map and we use Nominatim OSM. I set up the Nominatim API and Nominatim UI and are now up and running. I just want to raise a discussion if it is possible on the created Nominatim to have Red Pin like this when searching specific addresses or using latitude and longitude:
![image](https://user-images.githubusercontent.com/110162696/181658536-b777c523-ea99-4866-8085-cddc80a8c0b3.png)

Upon checking on openstreepmap.org, it has a feature like that:

![image](https://user-images.githubusercontent.com/110162696/181658737-cc4ec1db-10cf-4937-a5b8-45b3b3ae1365.png)

But on our Nominatim OSM, it doesn't have:

![image](https://user-images.githubusercontent.com/110162696/181658801-16f9ce4d-2918-42bb-ac74-cc92472edd82.png)

Thank you!

**Comment by mtmail** [comment]
https://github.com/osm-search/nominatim-ui is meant as a debugging interface for Nominatim server administrators. That's why it features 'debug URL' links, the 'show map bounds' contains a lot of extra data and clicking 'details' on a results gives more information that would overwhelm end users. You'd need to change your local installation to customize it.

https://github.com/osm-search/nominatim-ui/pull/187 you see hints how the circle gets created. It's a circle because we have two location we highlight on the map: the search position and the result position (end users usually are not interested in the search position).

Instead of `circle = L.circleMarker` try `L.Marker` (https://leafletjs.com/reference.html#marker)

**Reply by knee11** [reply]
Thank you for this sir!
